guides/module.yaml

```
@@ -14,6 +13,5 @@ DESCRIPTION
     
     To learn more about managing modules in a project, see the documentation.
     To learn how modules are loaded by Bolt, see the 'modulepath' guide.
-
-DOCUMENTATION
-    https://pup.pt/bolt-modules
+documentation:
```

```suggestion

documentation:
```

Just a small nit for consistency, since the rest of the guides have an extra newline before documentation. 😬